django/core/management/base.py

```diff
@@ -326,16 +328,16 @@ def run_from_argv(self, argv):
         handle_default_options(options)
         try:
             self.execute(*args, **cmd_options)
-        except Exception as e:
-            if options.traceback or not isinstance(e, CommandError):
```

Comment:
I'm not crazy - this is `isinstance` was redundant right?

Reply:
This is also an unrelated cleanup that should be moved to a separate commit.